wp-content/themes/core/templates/single.html

```diff
@@ -1,69 +1,3 @@
 <!-- wp:template-part {"slug":"header","tagName":"header","className":"site-header"} /-->
-<!-- wp:group {"tagName":"main"} -->
-<main class="wp-block-group">
-	<!-- wp:post-featured-image /-->
-	<!-- wp:post-title /-->
-	<!-- wp:post-author {"showAvatar":false} /-->
-	<!-- wp:post-date /-->
-	<!-- wp:post-terms {"term":"category"} /-->
-	<!-- wp:spacer {"height":40} -->
-	<div style="height:40px" aria-hidden="true" class="wp-block-spacer"></div>
-	<!-- /wp:spacer -->
-	<!-- wp:post-content {"layout":{"type":"constrained"}} /-->
-	<!-- wp:spacer {"height":40} -->
-	<div style="height:40px" aria-hidden="true" class="wp-block-spacer"></div>
-	<!-- /wp:spacer -->
-	<!-- wp:post-terms {"term":"post_tag"} /-->
-	<!-- wp:spacer {"height":40} -->
-	<div style="height:40px" aria-hidden="true" class="wp-block-spacer"></div>
-	<!-- /wp:spacer -->
-	<!-- wp:post-navigation-link {"type":"previous"} /-->
-	<!-- wp:post-navigation-link /-->
-	<!-- wp:spacer {"height":40} -->
-	<div style="height:40px" aria-hidden="true" class="wp-block-spacer"></div>
-	<!-- /wp:spacer -->
-	<!-- wp:heading {"className:"comments-title"} -->
-	<h2 class="comments-title">Comments</h2>
-	<!-- /wp:heading -->
-<!-- wp:comments-query-loop -->
-<div class="wp-block-comments-query-loop">
-	<!-- wp:comment-template -->
-	<!-- wp:columns -->
-	<div class="wp-block-columns"><!-- wp:column {"width":"40px"} -->
-	<div class="wp-block-column" style="flex-basis:40px">
-	<!-- wp:avatar {"size":40,"style":{"border":{"radius":"20px"}}} /--></div>
-	<!-- /wp:column -->
-	
-	<!-- wp:column -->
-	<div class="wp-block-column">
-	<!-- wp:comment-author-name /-->
-	
-	<!-- wp:group {"style":{"spacing":{"margin":{"top":"0px","bottom":"0px"}}},"layout":{"type":"flex"}} -->
-	<div class="wp-block-group" style="margin-top:0px;margin-bottom:0px">
-	<!-- wp:comment-date /-->
-	
-	<!-- wp:comment-edit-link /-->
-	</div>
-	<!-- /wp:group -->
-	
-	<!-- wp:comment-content /-->
-	
-	<!-- wp:comment-reply-link /-->
-	</div>
-	<!-- /wp:column --></div>
-	<!-- /wp:columns -->
-	<!-- /wp:comment-template -->
-	
-	<!-- wp:comments-pagination -->
-	<!-- wp:comments-pagination-previous /-->
-	
-	<!-- wp:comments-pagination-numbers /-->
-	
-	<!-- wp:comments-pagination-next /-->
-	<!-- /wp:comments-pagination -->
-	</div>
-	<!-- /wp:comments-query-loop -->
-	<!-- wp:post-comments-form /-->
-</main>
-<!-- /wp:group -->
-<!-- wp:template-part {"slug":"footer","tagName":"footer","className":"site-footer"} /-->
\ No newline at end of file
+<!-- wp:post-content {"align":"full","layout":{"type":"constrained"}} /-->
```

Comment:
🔴 same here, we don't want to lose the `main` element

Reply:
```suggestion
<!-- wp:group {"tagName":"main"} -->
<main class="wp-block-group">
	<!-- wp:post-content {"align":"full","layout":{"type":"constrained"}} /-->
</main>
<!-- /wp:group -->
```

@GeoffDusome maybe I'm missing something but I just tested this in my browser- if we don't filter the post content block to make it a `main` tag, and _instead_ just wrap the post content block like this the layout, padding, etc all looks the same.

The key is that the `main` group block wrapper can't have layout constrained. `.has-global-padding` only gets added to constrained blocks. Since we're constraining the post content block, we don't need `main` to be constrained and can therefore avoid a bunch of nested `.has-global-padding` containers causing layout issues.

Filtering seems like an over-engineered solution if we can just make the update in the template file. Let me know what you think.

Reply:
Agreed. Updated to match the above for both Post & Page.